packages/desktop-gui/cypress/fixtures/config.json

```diff
@@ -6,23 +6,23 @@
     {
       "name": "chrome",
       "displayName": "Chrome",
-      "family": "chrome",
+      "family": "chromium",
       "version": "78.0.3904.108",
       "path": "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
       "majorVersion": "78"
     },
     {
       "name": "chromium",
       "displayName": "Chromium",
-      "family": "chrome",
+      "family": "chromium",
       "version": "74.0.3729.0",
       "path": "/Applications/Chromium.app/Contents/MacOS/Chromium",
       "majorVersion": "74"
     },
     {
-      "name": "canary",
-      "displayName": "Canary",
-      "family": "chrome",
+      "name": "chrome",
+      "displayName": "Chrome Canary",
```

Comment:
This will break our recent updates to the browser icons in both the Test Runner and the Dashboard as we are relying on this name being `Canary`. https://github.com/cypress-io/cypress/pull/6230/files#diff-595682099de79f4c49ddf2630cdda0b6R8

Reply:
Yeah, this change will need to be coordinated with the services team. I noticed some tests failing for the browser icons in the desktop-gui, was going to look into that as soon as the 4.0 branch is passing so I can tell what's really failing.

Reply:
I don't think we want to change the `displayName` to be `Chrome Canary`... its for users, let's just leave it at `Canary` yeah ?

Reply:
Google appears to call it "Chrome Canary" everywhere it fits, only using "Canary" where it's space-limited, so that's why I updated it. I can change it back.

Dashboard will still need updates if they rely on `name === 'canary'` anywhere.

Reply:
Reverted the name back to "Canary". It seems like the dashboard relies on displayName, if so, then there should be no issues.

Reply:
After looking at the source code, the only browser data sent to the dashboard is the `displayName` and `version`:

https://github.com/cypress-io/cypress/blob/124d5904bd84f17e9fb8aae1b69c4708fefacc8d/packages/server/lib/modes/record.js#L614-L615

So since neither of those change in this PR, we're good :+1: